app/models/persistent_attachments/va_form.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class PersistentAttachments::VAForm < PersistentAttachment
+  include ::FormUpload::Uploader::Attachment.new(:file)
+
+  before_destroy(:delete_file)
+
+  CONFIGS = Hash.new(
+    { max_pages: 10, min_pages: 1 }
+  ).merge(
+    {
+      '21-0779' => { max_pages: 4, min_pages: 2 }
+    }
+  )
+
+  def max_pages
+    CONFIGS[form_id][:max_pages]
+  end
+
+  def min_pages
+    CONFIGS[form_id][:min_pages]
+  end
```

Comment:
A better way to do this might be:
```suggestion
  CONFIGS = Hash.new(
    { 
      default: { max_pages: 10, min_pages: 1 } },
      '21-0779' => { max_pages: 4, min_pages: 2 }
    }
  )

  def max_pages
    CONFIGS.fetch(form_id, :default)[:max_pages]
  end

  def min_pages
   CONFIGS.fetch(form_id, :default)[:min_pages]
  end
```